projects/experimental/components/badged-content/badged-content.style.less

```diff
@@ -0,0 +1,35 @@
+@import 'taiga-ui-local';
+
+:host {
+    --t-badge-height: 0;
+    --t-radius: 0.75rem;
```

Comment:
`--tui-radius`? If it's a public API var. Also it looks like it should default to `var(--tui-radius-l)`.